README.md

```diff
@@ -134,6 +119,25 @@ You may use CPU-specific optimization flags (-march=prescott, -march=i486, -marc
 as well as general optimization flags (-O1 and -O2), but DON'T USE -O3 flag, as Bullet tends to
 crash with this optimization level (GCC 5.1+ may compile it without errors).
 
+### Running and Configuration ###
+To run OpenTomb, simply run the executable generated by the build. By default, no command line options 
+are needed. Access the console by pressing `. This allows you to enter commands to select levels, change 
+settings, and more. Enter 'help' to get a list of commands. Enter 'exit' to quit the engine.
+
+Currently, all settings in OpenTomb are managed through config.lua and autoexec.lua.
+Config.lua contains persistent engine and game settings, while autoexec.lua contains
+any commands which should be executed on engine start-up.
+
+Config.lua is divided into different sections: screen, audio, render, controls, console and system. 
+In each of these sections, you can change numerous parameters, the names of which are usually fairly intuitive.  
+
+Autoexec.lua is a simple list of commands which are ran at startup. Modifying existing commands may cause the engine to
+function incorrectly.
+
+To select a level, enter 'setgamef(game, level) into either autoexec.lua or in the console, where game is 1-5. Mansion 
```

Comment:
Possibly the biggest issue I had getting the engine running was figuring out what to do to select a level. It might just have been me, but I only found out when looking at an issue somewhere. Others seem to have had the same issue, so this seems relevant to be here.